PIL/Image.py

```diff
@@ -1814,6 +1893,7 @@ def transform(x, y, matrix):
         return self.transform((w, h), AFFINE, matrix, resample)
 
     def save(self, fp, format=None, **params):
+        # type: (Union[Text, pathlib.Path, BytesIO], Optional[Text], **Any) -> None
```

Comment:
Is that as the filename to open or the file object to 'write' to? StringIO is unicode, at least by 2.6/7?